README.md

```diff
@@ -1,15 +1,5 @@
 # Jbuilder
 
-[![Build Status](https://api.travis-ci.org/rails/jbuilder.svg)][travis]
-[![Gem Version](http://img.shields.io/gem/v/jbuilder.svg)][gem]
-[![Code Climate](http://img.shields.io/codeclimate/github/rails/jbuilder.svg)][codeclimate]
-[![Dependencies Status](http://img.shields.io/gemnasium/rails/jbuilder.svg)][gemnasium]
-
-[travis]: https://travis-ci.org/rails/jbuilder
-[gem]: https://rubygems.org/gems/jbuilder
-[codeclimate]: https://codeclimate.com/github/rails/jbuilder
-[gemnasium]: https://gemnasium.com/rails/jbuilder
-
```

Comment:
Mainly because they are not useful for the final users.

* Travis badge don't add anything since we expect the build to be always green.
* Rubygems badge only give the last release, they can get the same information form GItHub release page.
* Codeclimate badge I don't even think codeclimate is useful, but even if it is gem users don't need to care about it
* Gemnasium it has useful informations only for contributors.

So, IMHO all badges are not useful for the users of the library but they may be useful for contributors so I think it belongs to the CONTRIBUTING.md file.